Makefile

```diff
@@ -122,8 +122,8 @@ ci:	## Run in automation environment
 	$(eval export AUTO_APPROVE=-auto-approve)
 	$(eval export SKIP_AZURE_LOGIN=true)
 
-.PHONY: terraform-app-init
-terraform-app-init: set-azure-account
+.PHONY: terraform-init
```

Comment:
tv has terraform-app and terraform-common. I think it's best to leave it that way. I would just add a terraform-destroy that points to terraform-app-destroy for the delete_review_app to work